apps/src/templates/teacherDashboard/SectionsPage.jsx

```diff
@@ -146,24 +120,25 @@ class SectionsPage extends Component {
   };
 
   render() {
-    const { numSections } = this.props;
-    const { sectionsLoaded } = this.state;
+    const { numSections, asyncLoadComplete } = this.props;
 
     const newSectionFlow = experiments.isEnabled(SECTION_FLOW_2017);
     const showGoogleClassroom = !newSectionFlow && this.provider === OAuthSectionTypes.google_classroom;
     const showCleverClassroom = !newSectionFlow && this.provider === OAuthSectionTypes.clever;
     return (
-      <div>
-        <div style={styles.breadcrumb}>
-          <a href="/teacher-dashboard#/">
-            {i18n.teacherHomePage()}
-          </a>
-          <span style={{opacity: 0.5}}>{"\u00a0 \u25b6 \u00a0"}</span>
-          <b style={{color: color.dark_orange}}>
-            {i18n.studentAccountsAndProgress()}
-          </b>
-        </div>
-        {sectionsLoaded &&
+      <div className={this.props.className}>
+        {!this.props.teacherHomepage &&
+          <div style={styles.breadcrumb}>
+            <a href="/teacher-dashboard#/">
+              {i18n.teacherHomePage()}
+            </a>
+            <span style={{opacity: 0.5}}>{"\u00a0 \u25b6 \u00a0"}</span>
+            <b style={{color: color.dark_orange}}>
+              {i18n.studentAccountsAndProgress()}
+            </b>
+          </div>
+        }
```

Comment:
Should we make this little breadcrumb a separate functional stateless component?